src/components/FacilityCard.vue

```diff
@@ -69,12 +73,13 @@
             <use xlink:href="#location" />
           </svg>
           <a
-            :href="`https://maps.apple.com/?daddr=${facility.city}+${facility.zip}+${facility.address}&dirflg=d`"
+            :href="`https://maps.apple.com/?daddr=${facility.address.city}+${facility.address.zip}+${facility.address.street}&dirflg=d`"
             target="_blank"
             class="icon-link"
             aria-label="Naviger til faciliteten"
           >
-            {{ facility.address }}, {{ facility.city }}, {{ facility.zip }}
+            {{ facility.address.street }}, {{ facility.address.city }},
+            {{ facility.address.zip }}
```

Comment:
Don't know why the format is like this but normally it's done like in the suggestion

```suggestion
            {{ facility.address.street }}, {{ facility.address.zip }} {{ facility.address.city }}
```

Reply:
This is the prettier formatting in relation to long lines. It is ugly agreed, but we should keep following the auto formatting.